web/src/components/Dialog.tsx

```diff
@@ -26,16 +27,28 @@ export default function AlertDialog(props: IProps) {
     <div>
```

Comment:
Do we need to have this encapsulating tag?

Reply:
This component exist previously with encapsulating tag. Moved it to functional component as rest part components.

Reply:
Thanks